### ISSUE TITLE
Location and format of L4 Ameriflux Observations have changed

### ISSUE THREAD

**Issue by braczka** [NCAR/DART#339]
### Describe the bug

The repository for the L4 Ameriflux data used for land modeling has changed and the current [documentation page for the L4 obs converter](https://docs.dart.ucar.edu/en/latest/observations/obs_converters/Ameriflux/level4_to_obs.html) data sources link is broken. ([broken link](http://cdiac.ornl.gov/ftp/ameriflux/data/Level4/Sites_ByName))

As far as i can tell the current repository has been migrated to here:  https://fluxnet.org/data/download-data/   or  https://ameriflux.lbl.gov/data/download-data/.     

This is really two issues:  the documentation needs to be updated, and the observation converter needs to be updated
to reflect the change in repository and change in file format.   

### Which model(s) are you working with?
Relevant to CLM or any land model using carbon, water land-atmosphere fluxes.

### Have you modified the DART code?   
I have not made any modifications to the ``~/observations/obs_converters/Ameriflux/level4_to_obs.f90`` code yet, but the file
format of the repository has changed, thus changes are required..

**Comment by hkershaw-brown** [comment]
Note for building the Ameriflux observation converter:

```
cd DART/observations/obs_converters/Ameriflux/work
./quickbuild.csh
```

Running the converter:
```
./level4_to_obs  
```

The converter reads a bunch of options from the file input.nml.  The level4_to_obs options are in the `&level4_to_obs_nml` section of the file:

```
input.nml

 26 &level4_to_obs_nml
 27    text_input_file = '../data/USHa12003_L4_h.txt'
 28    obs_out_file    = 'obs_seq.out'
 29    year            = 2003
 30    timezoneoffset  = -6
 31    latitude        =  42.5378
 32    longitude       = -72.1715
 33    elevation       = 353
 34    flux_height     = 29
 35    maxgoodqc       = 3
 36    verbose         = .TRUE.
 37    /
```

**Comment by hkershaw-brown** [comment]
@braczka @nancycollins  Are there any of the old format files hanging around, e.g.  '../data/USHa12003_L4_h.txt' ? Maybe on Cheyenne?

**Comment by braczka** [comment]
I did not locate any of these files the last time I looked at this -- but I didn't give it a thorough look because I knew the file formatting had likely changed.  I agree, It would be helpful to have these to test the existing obs_converter though......will see if I can locate some.

**Comment by hkershaw-brown** [comment]
@mjs2369  added a Makefile to
https://ncar.github.io/dart-developers-guide/documentation.html
in case you want to build the documentation on your machine with Sphinx

**Comment by nancycollins** [comment]
sorry - i didn't ever work with the ameriflux converter, and don't know of any input files.

**Comment by nancycollins** [comment]
i did a bit of poking around.  i did not find any input data, but in the observations/obs_converters/Ameriflux/shell_scripts is a script that creates a series of obs_seq files from what i presume was Ameriflux data.  the output files are on cheyenne, here:

/glade/p/cisl/dares/Observations/FluxTower

this is based on the filenames matching up with the names used in the script.  they might be useful in validating any input files that are found.

**Comment by braczka** [comment]
Thank you Nancy -- I did a bit of poking around as well, and apparently the old L4 style format is still hosted by the University of Montana Terrasystems group.   I downloaded a few of the test files here if you want them: /glade/scratch/bmraczka/AmerifluxL4_oldformat

**Comment by nancycollins** [comment]
i believe this is going to be marlee's project, so i'm going to back away slowly and not make eye contact!!  :)

**Comment by braczka** [comment]
Yes !  Agreed -- Marlee was assigned to this fun project :-)

**Comment by mjs2369** [comment]
Thank you, Brett and Nancy, for locating those files for me! I will look into it :)

**Comment by braczka** [comment]
I have been able to make some headway on other site-level land DA PR's (e.g. #461) and have circled back to this one.  It would be useful to have an updated Ameriflux L4 obs converter given interest in site level land DA work, including state and parameter estimation using CLM.   Given the complexity of eddy covariance data with the new Ameriflux  format (e.g. temporal resolution, uncertainty, QA/QC) it makes sense for me to lead this project.  (I have a lot of experience working with land-atmosphere flux data).   I will start working on a PR in near future, and will assign @mjs2369 to review to help with the software implementation.

**Comment by mjs2369** [comment]
@braczka sounds great, thanks for taking the lead on this

**Comment by braczka** [comment]
This issue fixed in #478 .
